terraform/deployments/monitoring-test/base/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  backend "s3" {
+    bucket  = "govuk-terraform-test"
```

Comment:
```suggestion
    bucket  = var.govuk_aws_state_bucket
```

Reply:
I can do that.

Reply:
not possible, see issue: https://github.com/hashicorp/terraform/issues/13022